components/head.js

```diff
@@ -1,12 +1,13 @@
-import Head from 'next/head'
+import Head from "next/head"
 
-export default function HeadObject({children}) {
-    const title = "The Perfect Next.js Theme";
-    const description = "This is the perfect Next.js theme that you should use instead of create-next-app.";
-    const keywords = "put some fun keywords here";
-    const author = "Sarthak Mohanty";
-    const twitter = "@sarthaktexas";
-    //const image = "/ogimage.png"; // This is your OpenGraph image
+export default function HeadObject({ children }) {
+    const title = "The Perfect Next.js Theme"
+    const description =
+        "This is the perfect Next.js theme that you should use instead of create-next-app."
```

Comment:
why is this on a new line?!?

Reply:
https://github.com/ThinkFiveable/open/blob/8fb8e0ba9c4750bf91c7e364d7dcb9ad137f0d03/.prettierrc#L3